acurl_ng/setup.cfg

```
@@ -0,0 +1,35 @@
+[metadata]
+name = acurl_ng
+description = An async Curl library.
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.9
+author = Tony Simpson
+author_email = agjasimpson@gmail.com
+maintainer = Sky Identity NFT team
+maintainer_email = aaron.ecay@sky.uk
```

```suggestion
author = Aaron Ecay
author_email = aaron.ecay@sky.uk
maintainer = Sky Identity NFT team
maintainer_email = matthew.ellis@sky.uk
```

Congratulations on your maintainership, btw :joy:

---

Updated 😄